### ISSUE TITLE
How do you divide the training set and test set?

### ISSUE THREAD

**Issue by cherwinsd** [me301/sun-attribute-classification#1]
There are only twenty to thirty images per category in the dataset, and I tried to train the classification model with resnet using an eight-to-two division ratio, and found that it was very easy to overfitting.
